Documentation/gettingstarted/clustermesh/clustermesh.rst

```diff
@@ -26,6 +30,28 @@ Cluster Addressing Requirements
 * The network between clusters must allow the inter-cluster communication. The
   exact ports are documented in the :ref:`firewall_requirements` section.
 
+Additional Requirements for Native-routed Datapath Modes
+--------------------------------------------------------
+
+* Cilium in each cluster must be configured with a native routing CIDR that
+  covers all the PodCIDR ranges across all connected clusters. Cluster CIDRs are
+  typically allocated from the ``10.0.0.0/8`` private address space. When this
+  is a case a native routing CIDR such as ``10.0.0.0/9`` should cover all
```

Comment:
@jrajahalme was there a reason that this lists `10.0.0.0/9` rather than `10.0.0.0/8`?